__tests__/fixtures/react-native/fabricRefRewrites/reactDevelopment/output.js

```diff
@@ -711,27 +711,20 @@ if (process.env.NODE_ENV !== 'production') {
       ];
       const isTurboModuleEnabled = global.RN$Bridgeless || global.__turboModuleProxy != null;
       if (isTurboModuleEnabled && Platform.OS === 'ios') {
-        if (
-          type.$$typeof &&
```

Comment:
The only change here is removing `type.$$typeof`. You'll see this being removed in every other file as well. This was added prior to RN 81, since during testing, every component that had FS Attributes contained `type.$$typeof`, and the check was added to avoid rewriting unnecessary components. `$$typeof` is no longer on `type` in RN 81, and simply checking whether `propContainsFSAttribute` should suffice in targeting any components we want to rewrite.